protostar/commands/test/test_command.py

```diff
@@ -152,6 +152,16 @@ def arguments(self):
                 type="bool",
                 description="Only re-run failed and broken test cases.",
             ),
+            ProtostarArgument(
```

Comment:
Do we need involve gas price? It seems not very intuitive, I think gas estimation is more than enough for relative assesment, I would presonally change it to `--estimate-gas` boolean to simplify it.

Reply:
I left the decision to the user. If the user wants to estimate gas, he can set `--gas-price 1`.